diff --git a/byte-buddy-dep/pom.xml b/byte-buddy-dep/pom.xml
index 3f8fa6217f..07ed398b33 100644
--- a/byte-buddy-dep/pom.xml
+++ b/byte-buddy-dep/pom.xml
@@ -48,10 +48,6 @@
                     <groupId>org.ow2.asm</groupId>
                     <artifactId>asm-analysis</artifactId>
                 </exclusion>
-                <exclusion>
-                    <groupId>org.ow2.asm</groupId>
-                    <artifactId>asm-tree</artifactId>
-                </exclusion>
             </exclusions>
         </dependency>
         <dependency>
diff --git a/byte-buddy-dep/src/main/java/net/bytebuddy/dynamic/scaffold/InstrumentedType.java b/byte-buddy-dep/src/main/java/net/bytebuddy/dynamic/scaffold/InstrumentedType.java
index 936a7c2b61..302c55e373 100644
--- a/byte-buddy-dep/src/main/java/net/bytebuddy/dynamic/scaffold/InstrumentedType.java
+++ b/byte-buddy-dep/src/main/java/net/bytebuddy/dynamic/scaffold/InstrumentedType.java
@@ -44,7 +44,8 @@ import static net.bytebuddy.matcher.ElementMatchers.is;
 import static net.bytebuddy.matcher.ElementMatchers.not;
 
 /**
- * Implementations of this interface represent an instrumented type that is subject to change. Implementations
+ * Implementations of this 
+  represent an instrumented type that is subject to change. Implementations
  * should however be immutable and return new instance when its builder methods are invoked.
  */
 public interface InstrumentedType extends TypeDescription {
@@ -509,10 +510,10 @@ public interface InstrumentedType extends TypeDescription {
          * A set containing all keywords of the Java programming language.
          */
         private static final Set<String> KEYWORDS = new HashSet<String>(Arrays.asList(
-                "abstract", "continue", "for", "new", "switch", "assert", "default", "goto", "package", "synchronized", "boolean",
+                "abstract", "continue", "for", "new", "switch", "assert", "goto", "package", "synchronized", "boolean",
                 "do", "if", "private", "this", "break", "double", "implements", "protected", "throw", "byte", "else", "import",
                 "public", "throws", "case", "enum", "instanceof", "return", "transient", "catch", "extends", "int", "short",
-                "try", "char", "final", "interface", "static", "void", "class", "finally", "long", "strictfp", "volatile",
+                "try", "char", "final", "static", "void", "class", "finally", "long", "strictfp", "volatile",
                 "const", "float", "native", "super", "while"
         ));
 
diff --git a/byte-buddy/pom.xml b/byte-buddy/pom.xml
index 35e8445b31..1c5bd75787 100644
--- a/byte-buddy/pom.xml
+++ b/byte-buddy/pom.xml
@@ -277,6 +277,7 @@
                                                 <exclude>**/NOTICE</exclude>
                                             </excludes>
                                         </filter>
+                                         <!-- 
                                         <filter>
                                             <artifact>org.ow2.asm:asm-commons</artifact>
                                             <includes>
@@ -292,6 +293,7 @@
                                                 <include>org/objectweb/asm/commons/SimpleRemapper.**</include>
                                             </includes>
                                         </filter>
+                                        -->
                                     </filters>
                                     <transformers>
                                         <transformer implementation="org.apache.maven.plugins.shade.resource.ManifestResourceTransformer">
